packages/cyberstorm/src/components/Button.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { MouseEventHandler } from "react";
+import styles from "./componentStyles/Button.module.css";
+
+export interface ButtonProps {
+  label?: string;
+  leftIcon?: string;
+  rightIcon?: string;
+  buttonStyle?: string;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+/**
+ * Cyberstorm Button component
+ */
+export const Button: React.FC<ButtonProps> = (props) => {
+  const { label, leftIcon, rightIcon, buttonStyle, onClick } = props;
+  const stylesImport = styles;
```

Comment:
I wonder what would be the best way to use the styles🤔 If the styles is not called, the styles don't seem to be imported. (even if the variable is never used... weird)

Reply:
Missed the comment earlier since the discussion was marked resolved by a bot.

Just writing the line 17 as `styles;` seems to work for me. Perhaps not the correct way to fix it, but at least it removes the error in my IDE. But do write an additional comment that states why the odd line exists, so the next dev doesn't waste time trying to remove it.

Reply:
Added a comment and switched to only `styles;`